external-docs/docs/quick-start/designing-csv.md

```diff
@@ -2,10 +2,10 @@
 
 csvcubed needs to understand how your statistical data is structured in order to make it more machine readable. There are two ways that you can do this with csvcubed:
 
-* Using the [convention-first approach](../guides/qube-config.md#convention-first-method) - you start out with a [standard CSV data shape](../guides/shape-data.md#standard-shape) with conventional column titles and fill it out with your data.
-* Using the [configuration approach](../guides/qube-config.md#configuration) - you write a JSON configuration file which tells csvcubed exactly how to interpret your data.
+* Using the [configuration by convention approach](../guides/configuration/convention.md) - you start out with a [standard CSV data shape](../guides/shape-data.md#standard-shape) with conventional column titles and fill it out with your data.
+* Using the [configuration approach](../guides/configuration/qube-config.md) - you write a JSON configuration file which tells csvcubed exactly how to interpret your data.
```

Comment:
It's quite possible we'll be renaming these things later. I think naming them this is better than the `convention-first` approach we had since the `-first` lost meaning given we apply the conventional approach as a last resort when we don't have any explicit configuration.

Perhaps `explicit configuration` is better than just `configuration`?